db/migrate/20160223172116_rolify_create_roles.rb

```diff
@@ -0,0 +1,19 @@
+class RolifyCreateRoles < ActiveRecord::Migration
+  def change
+    create_table(:roles) do |t|
+      t.string :name
+      t.references :resource, :polymorphic => true
+
+      t.timestamps
+    end
+
+    create_table(:users_roles, :id => false) do |t|
+      t.references :user
+      t.references :role
+    end
```

Comment:
I can change the hash syntax if you'd like, this file was generated entirely by the Rolify gem as part of its setup instructions, so I didn't go out of my way to modify it.